src/model.ts

```diff
@@ -7,67 +7,124 @@ import { IContentFieldFilter, IContentSearchRequest } from '@esri/hub-search';
 import { PassThrough } from 'stream';
 import { PagingStream } from './paging-stream';
 import { getBatchedStreams } from './helpers/get-batched-streams';
-import { fetchSite, getHubApiUrl, getPortalApiUrl, hubApiRequest, RemoteServerError } from '@esri/hub-common';
+import { fetchSite, getHubApiUrl, getPortalApiUrl, hubApiRequest, IModel, RemoteServerError } from '@esri/hub-common';
```

Comment:
check that you are using the most up to date hub-common.  It looks like there is a new method: https://esri.github.io/hub.js/api/common/fetchSiteModel/